atlp-rwanda/cod-be
# #181414545 Fix: configure ESLint

### What does this PR do?
This PR configures ESLint in the project and replaces the previous branch which had changes from all the collaborators.

### Description of Task to be completed?

- [x] Define rules to be followed in `.eslintrc` file

- [x] Add npm script to lint the project

### How should this be manually tested?

-   Navigate to the terminal and run `npm i` to install ESlint and Prettier packages
-  run `npm lint` to lint the project and check if there any rules violated 
- If you want to fix the fixable errors that were found you can run `npm run lint:fix`
- You can also run `npm run prettier` to check if your branch adhere to predefined rules.
-  If you find you can automatically format the files by running `npm run prettier:fix` 
-  Or if you want to do them at the same time run `npm format` which run `npm run prettier:fix && npm run lint:fix`
